openslides_backend/action/actions/motion/create.py

```diff
@@ -103,78 +83,9 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
-                workflow_id = meeting.get("motions_default_amendment_workflow_id")
-            elif instance.get("statute_paragraph_id"):
-                workflow_id = meeting.get(
-                    "motions_default_statute_amendment_workflow_id"
-                )
-            else:
-                workflow_id = meeting.get("motions_default_workflow_id")
-        if workflow_id:
-            workflow = self.datastore.get(
-                FullQualifiedId(Collection("motion_workflow"), workflow_id),
-                ["first_state_id"],
-            )
-            instance["state_id"] = workflow.get("first_state_id")
-        else:
-            raise ActionException(
-                "No matching default workflow defined on this meeting"
-            )
-
-        # check for origin_id
-        if instance.get("origin_id"):
-            meeting = self.datastore.get(
-                FullQualifiedId(Collection("meeting"), instance["meeting_id"]),
-                ["committee_id"],
-            )
-            forwarded_from = self.datastore.get(
-                FullQualifiedId(Collection("motion"), instance["origin_id"]),
-                ["meeting_id"],
-            )
-            forwarded_from_meeting = self.datastore.get(
-                FullQualifiedId(Collection("meeting"), forwarded_from["meeting_id"]),
-                ["committee_id"],
-            )
-            committee = self.datastore.get(
-                FullQualifiedId(
-                    Collection("committee"), forwarded_from_meeting["committee_id"]
-                ),
-                ["forward_to_committee_ids"],
-            )
-            if meeting["committee_id"] not in committee.get(
-                "forward_to_committee_ids", []
-            ):
-                raise ActionException(
-                    f"Committee id {meeting['committee_id']} not in {committee.get('forward_to_committee_ids', [])}"
-                )
-
-        # create submitters
-        submitter_ids = instance.pop("submitter_ids", None)
-        if not submitter_ids:
-            submitter_ids = [self.user_id]
-        self.apply_instance(instance)
-        action_data = []
-        for user_id in submitter_ids:
-            action_data.append({"motion_id": instance["id"], "user_id": user_id})
-        self.execute_other_action(MotionSubmitterCreateAction, action_data)
-
-        instance["sequential_number"] = self.get_sequential_number(
-            instance["meeting_id"]
-        )
-        # set created and last_modified
-        timestamp = round(time.time())
-        instance["created"] = timestamp
-        instance["last_modified"] = timestamp
-        self.set_number(
-            instance,
-            instance["meeting_id"],
-            instance["state_id"],
-            instance.get("lead_motion_id"),
-            instance.get("category_id"),
-        )
-
+        self.set_state_from_workflow(instance, meeting)
+        self.check_for_origin_id(instance)
```

Comment:
There is not `origin_id` settable for motion.create. This can be just implemented in the create_forwarded class

Reply:
Ah ok. I will remove this from [the wiki](https://github.com/OpenSlides/OpenSlides/wiki/motion.create) then.

Reply:
@reiterl please remove from the action. Then this can be merged

Reply:
Remove origin_id from motion.create and update tests.